integration-tests/api/__tests__/admin/discount.js

```diff
@@ -1244,6 +1263,55 @@ describe("/admin/discounts", () => {
           )
         })
     })
+
+    it("fails to create a discount if regions array contains an invalid regionID ", async () => {
```

Comment:
suggestion: `fails to create a discount if the regions contains an invalid regionId`